### PR TITLE
Fix/changelog

### PR DESCRIPTION
**Pull request by bacarvalho**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [x] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/changelog/current-year.es.md [hunk 1]
```diff
@@ -14,4 +14,29 @@ Cada release note describe los cambios que aplican a una versión. Estos cambios
 
 Si tienes campañas de publicidad para tu negocio, es importante que puedas seguirlas y ver si te están ayudando a concretar ventas. Para poder mejorarlas y que sean cada más eficientes, sumamos la posibilidad de asociar un píxel de Facebook y una etiqueta de seguimiento de conversiones de Google Ads a los pagos de tu Checkout de Mercado Pago.
 
-[Comienza a medir la conversión de tus anuncios](https://www.mercadopago.com.ar/developers/es/guides/payments/web-payment-checkout/configurations/).
\ No newline at end of file
+[Comienza a medir la conversión de tus anuncios](https://www.mercadopago.com.ar/developers/es/guides/payments/web-payment-checkout/configurations/).
+
+
+## 27 de marzo de 2019
```
Comment (pamalv7) [inline guides/changelog/current-year.es.md]:
```suggestion
## 27 de marzo de 2020
```

File: guides/changelog/current-year.pt.md [hunk 2]
```diff
@@ -14,4 +14,31 @@ Cada release note descreve as alterações que se aplicam a uma versão. Essas a
 
 Se você possui campanhas publicitárias para seu negócio, é importante acompanhá-las e ver se elas estão lhe ajudando a concretizar vendas. Para poder melhorá-las e sejam cada vez mais eficientes, acrescentamos a possibilidade de associar um pixel do Facebook e uma tag de acompanhamento de conversões do Google Ads aos pagamentos de seu Checkout Mercado Pago.
 
-[Começar a medir a conversão de seus anúncios](https://www.mercadopago.com.ar/developers/es/guides/payments/web-payment-checkout/configurations/).
\ No newline at end of file
+[Começar a medir a conversão de seus anúncios](https://www.mercadopago.com.ar/developers/es/guides/payments/web-payment-checkout/configurations/).
+
+
+## 27 de março de 2019
```
Comment (pamalv7) [inline guides/changelog/current-year.pt.md]:
```suggestion
## 27 de março de 2020
```

File: guides/changelog/current-year.en.md [hunk 3]
```diff
@@ -14,4 +14,30 @@ Each release note describes the new changes applying to a version. This changes
...
+
+
+## March 27th, 2020
+ 
+In order to help you optimize your reconciliations, we made some changes in the generation of the date ranges for your scheduled reports. This change applies to both the Available Balance report and Account Balance report. After April 1st, 2020 to find your scheduled reports, you will have to search for them by one second less. 
+
+| `Search` Filters | Actual Date | New Date |
+| :--------- | :------------------------ | :------------------------------- |
+begin_date  | 01/01/2020 00:00:00 | 01/01/2020 00:00:00
+end_date | 02/01/2020 |  00:00:00 01/01/2020  23:59:59
```
Comment (nicolassalvanes) [inline guides/changelog/current-year.en.md]: la hora de Actual Date quedó en la columna de New Date

File: guides/changelog/current-year.pt.md [hunk 4]
```diff
@@ -14,4 +14,27 @@ Cada release note descreve as alterações que se aplicam a uma versão. Essas a
...
+Seu novo parâmetro ficaria da seguinte forma: 
+
+https://api.mercadolibre.com/account/bank_report/search?access_token={{access_token}}
+	&created_from=schedule
+	&user_id=290477154
+	&begin_date=2020-01-01T00:00:00Z
+	&end_date=2020-01-01T23:59:59Z
+
+
+> Para mais informações sobre a programação de relatórios, [onsulte a documentação](https://www.mercadopago.com.ar/developers/pt/guides/reports/general-considerations/reconciliation-reports/).
```
Comment (pamalv7) [inline guides/changelog/current-year.pt.md]:
```suggestion
> Para mais informações sobre a programação de relatórios, [consulte a documentação](https://www.mercadopago.com.ar/developers/pt/guides/reports/general-considerations/reconciliation-reports/). 
```

File: guides/changelog/current-year.en.md [hunk 5]
```diff
@@ -14,4 +14,26 @@ Each release note describes the new changes applying to a version. This changes
...
+
+
+## March 27th, 2020
+ 
+In order to help you optimize your reconciliations, we made some changes in the generation of the date ranges for your scheduled reports. This change applies to both the Available Balance report and Account Balance report. After April 1st, 2020 to find your scheduled reports, you will have to search for them by one second less. 
+
+ `Search` Filters | Actual Date | New Date |
+--------- | ------------------------ | ------------------------------- |
+begin_date  | 01/01/2020 00:00:00 | 01/01/2020 00:00:00
+end_date | 02/01/2020 |  00:00:00 01/01/2020  23:59:59
```
Comment (Juanii95) [inline guides/changelog/current-year.en.md]:
```suggestion
end_date | 02/01/2020 00:00:00 |  01/01/2020  23:59:59
```

File: guides/changelog/current-year.es.md [hunk 6]
```diff
@@ -14,4 +14,26 @@ Cada release note describe los cambios que aplican a una versión. Estos cambios
...
+
+
+## 27 de marzo de 2020
+
+Para ayudarte a optimizar tus conciliaciones, implementamos una mejora en la generación de los rangos de fechas para reportes programados. El cambio aplica tanto para el reporte de Dinero retirado, como al reporte de Todas las transacciones. Desde el miércoles 1 de abril, para encontrar tus reportes programados, vas a tener que buscarlos por un segundo menos. 
+
+ Filtros del `search`| Fecha atual | Nueva fecha |
+--------- | ------------------------ | ------------------------------- |
+begin_date  | 01/01/2020 00:00:00 | 01/01/2020 00:00:00
+end_date | 02/01/2020 |  00:00:00 01/01/2020  23:59:59
```
Comment (Juanii95) [inline guides/changelog/current-year.es.md]:
```suggestion
end_date | 02/01/2020  00:00:00 | 01/01/2020  23:59:59
```

File: guides/changelog/current-year.pt.md [hunk 7]
```diff
@@ -14,4 +14,27 @@ Cada release note descreve as alterações que se aplicam a uma versão. Essas a
...
+
+## 27 de março de 2020
+
+Para ajudar a otimizar suas conciliações, implementamos melhorias na geração dos intervalos de datas para relatórios programados. A mudança é válida tanto para o relatório de Dinheiro retirado, como para o relatório de Todas as transações. Desde quarta-feira 1 de abril, para encontrar seus relatórios programados, busque-os por um segundo a menos.
+
+
+Filtros do `search`| Data atual | Nova data |
+ --------- | ------------------------ | ------------------------------- |
+begin_date  | 01/01/2020 00:00:00 | 01/01/2020 00:00:00
+end_date | 02/01/2020 |  00:00:00 01/01/2020  23:59:59
```
Comment (Juanii95) [inline guides/changelog/current-year.pt.md]:
```suggestion
end_date | 02/01/2020  00:00:00 | 01/01/2020  23:59:59
```

**Pull request merged**